action.yml

```
@@ -19,16 +19,21 @@ inputs:
   file_pattern:
     description: 'The file glob e.g. pages/**/*.mdx'
     required: true
+  default_branch:
+    description: 'The default branch in the target repository'
+    default: 'master'
+    required: false
 outputs:
   inspec-output:
     description: 'The output of inspec'
 runs:
   using: 'docker'
-  image: 'docker://hashieducation/inspec' 
+  image: 'docker://docker.mirror.hashicorp.services/hashieducation/inspec:ks.adds-default-branch-env'
```

```suggestion
  image: 'docker://docker.mirror.hashicorp.services/hashieducation/inspec:master'
```
Need to update this after we push to docker hub and sync to mirror and before we merge this PR.